rp2040-hal/Cargo.toml

```diff
@@ -39,6 +39,7 @@ chrono = { version = "0.4", default-features = false, optional = true }
 defmt = { version = ">=0.2.0, <0.4", optional = true }
 
 rtic-monotonic = { version = "1.0.0", optional = true }
+num-traits = { version = "0.2.15", default-features = false }
```

Comment:
I think this isn't used at all. At least clippy marks the only obvious reference to this crate, the import of `FloatCore`, as unused.